src/node_perf_common.h

```diff
@@ -76,7 +77,10 @@ class performance_state {
       isolate,
       offsetof(performance_state_internal, observers),
       NODE_PERFORMANCE_ENTRY_TYPE_INVALID,
-      root) {}
+      root) {
+    for (size_t i = 0; i < milestones.Length(); i++)
+      milestones[i] = -1.;
```

Comment:
Is there a reason this is set to `-1.` and not `-1`? I'm far from a C++ expert so would love to better understand what's going if there is. Thanks!

Reply:
They are one and the same in this case. -1 (an integer literal) will get converted to a double anyway. I just wanted to make it explicit that we are assigning a floating point value -1 here.

Reply:
I figured that might've been the intention but wanted to make sure. Thanks.